README.md

```diff
@@ -1,11 +1,14 @@
 # Instance Scheduler on AWS (ID SO0030)
 
-Scheduler for Cross-Account and Cross-Region scheduling for EC2 and RDS instances
+Instance Scheduler on AWS automates the starting and stopping of Amazon Elastic Compute Cloud (Amazon EC2) and Amazon Relational Database Service (Amazon RDS) instances.
+
+This solution helps reduce operational costs by stopping resources that are not in use and starting them when they are needed. The cost savings can be significant if you leave all of your instances running at full utilization continuously.
```

Comment:
suggest rephrasing: "The cost savings can be significant _compared to_ leaving instances running continuously."

Reply:
phrasing is (mostly) copy-pasted from solution landing page.